### ISSUE TITLE
Updating Rancher desktop on M1 mac installs the x86 version

### ISSUE THREAD

**Issue by frob** [rancher-sandbox/rancher-desktop#1162]
### Rancher Desktop Version

0.7.1

### Rancher Desktop K8s Version

unknown

### What operating system are you using?

macOS

### Operating System / Build Version

12.0.1

### What CPU architecture are you using?

arm64 (Apple Silicon)

### Actual Behavior

I installed pressed the update button on Rancher Desktop and Rancher downloaded the update and when it restarted it gave me the error "This version of Rancher desktop is for x86"

<img width="940" alt="screenshot of error" src="https://user-images.githubusercontent.com/316176/147421670-ff709dbe-1925-4bc1-b7e6-0f93f2adb1ac.png">

### Steps to Reproduce

install 0.7.0 arch64 version
open rancher desktop
update rancher desktop
restart rancher desktop
Notice error

### Result

"Fatal Error" and the program halts.

<img width="940" alt="" src="https://user-images.githubusercontent.com/316176/147421706-ea89aa35-80ec-4386-873e-22641521253b.png">

### Expected Behavior

A successful update of rancher desktop.

### Additional Information

I never got it to run in the first place. I always got the Lima failed to start error but that is another issue.

**Comment by mook-as** [comment]
Reproducing the issue:
- Download [0.7.0 darwin arm64](https://github.com/rancher-sandbox/rancher-desktop/releases/download/v0.7.0/Rancher.Desktop-0.7.0.aarch64.dmg).
- Start it (updating checking was disabled from a previous install)
- Enable update checking.
- Updater logs claim to have found the aarch64 file (https://github.com/rancher-sandbox/rancher-desktop/releases/download/v0.7.1/Rancher.Desktop-0.7.1-mac.aarch64.zip)
- Restarting the app, it updated to version 0.7.1
- The cluster is running correctly.

That is to say, I can't seem to reproduce the issue (I typed bits of the steps as I did them).  I do notice that we don't show which arch we're running in the app at all, though; that might be helpful (in case we somehow started with the wrong version, etc.).
